src/docs/sdk/features.mdx

```diff
@@ -231,6 +231,58 @@ If Performance Monitoring is both supported by the SDK and enabled in the client
 - span status must match HTTP response status code ([see Span status to HTTP status code mapping](/sdk/event-payloads/span/))
 - when network error occurs, span status must be set to `internal_error`
 
+### HTTP Client errors
+
+HTTP Client errors are automatically captured and sent to `https://sentry.io`.
+
+The HTTP Client integration should have 3 configuration options:
```

Comment:
`m`: On Cocoa, we don't expose integration options. We use the global SentryOptions for everything. Is this a typical pattern more SDKs use? If yes, we should also add it to Cocoa then. If not, I think we should change to option names, so they reflect they relate to the HTTP client error integration.

Reply:
The difference is that Cocoa that thru Swizzling so no need of adding interceptors manually, here it is needed so it makes more sense to be per integration.
I can have an HTTP Client that captures events and the other one don't because I don't want to, a global config in the SentryOptions would not allow me to do that.

Reply:
We will use swizzling, so most likely, we will add this to the global options. But I think the names should also be fine on the global options. They are kind of clear. We only need proper code docs.